doc/book/connectors/__python.rst

```diff
@@ -39,8 +39,181 @@ For an example of using Python API with
 * `gtarantool <https://github.com/shveenkov/gtarantool>`__ with gevent support **no active maintenance**
+
+Feature comparison
+------------------
+
+Last update: January 2022
+
+..  list-table::
+    :header-rows: 1
+    :stub-columns: 1
+
+    *   -   Parameter
+        -   `igorcoding/asynctnt <https://github.com/igorcoding/asynctnt>`__
+        -   `shveenkov/gtarantool <https://github.com/shveenkov/gtarantool>`__
+        -   `tarantool/tarantool-python <https://github.com/tarantool/tarantool-python>`__
+
+    *   -   License
+        -   Apache License 2.0
+        -   LGPL
+        -   BSD-2
+
+    *   -   Is maintained
+        -   Yes
+        -   No (last updated in 2018)
+        -   Yes
+
+    *   -   Known Issues
+        -   `issue #18 <https://github.com/igorcoding/asynctnt/issues/18>`__ (no running event loop)
+        -   None
+        -   `issue #105 <https://github.com/tarantool/tarantool-python/issues/105>`__ (Unpack of binary data)
+
+    *   -   Documentation
+        -   Yes (`github.io <https://igorcoding.github.io/asynctnt/>`__)
+        -   No
+        -   Yes (`tarantool.io
+            <https://www.tarantool.io/en/doc/latest/getting_started/getting_started_python/>`__
+            and `readthedocs
+            <https://tarantool-python.readthedocs.io/en/latest/quick-start.en.html>`__
+            (`obsolete
+            <https://github.com/tarantool/tarantool-python/issues/67>`__))
+
+    *   -   Testing / CI / CD
+        -   Github Actions
+        -   No (tests exists)
+        -   Appveyor (only Windows)
+
+    *   -   Github Stars
+        -   51
+        -   17
+        -   78
+
+    *   -   Static Analysis
+        -   Yes (Flake8)
+        -   No
+        -   No
+
+    *   -   Packaging
+        -   `Pip <https://pypi.org/project/asynctnt/>`__
+        -   `Pip <https://pypi.org/project/gtarantool/>`__
+        -   `Deb, RPM, Pip <https://github.com/tarantool/tarantool-python#download-and-install>`__
+
+    *   -   Code coverage
+        -   Yes
+        -   No
+        -   Yes
+
+    *   -   Support asynchronous mode
+        -   Yes, `asyncio <https://docs.python.org/3/library/asyncio.html>`__
+        -   Yes (`gevent
+            <https://www.gevent.org/api/gevent.event.html#gevent.event.AsyncResult>`__,
+            example: `test_gevent.py
+            <https://github.com/shveenkov/gtarantool/blob/master/tests/test_gevent.py>`__)
+        -   No
+
+    *   -   Batching support
+        -   No
+        -   No
+        -   No (`issue #55 <https://github.com/tarantool/tarantool-python/issues/55>`__)
+
+    *   -   Schema reload
+        -   Yes (automatically, see `auto_refetch_schema <https://igorcoding.github.io/asynctnt/api.html>`__)
+        -   Yes (automatically)
+        -   Yes (automatically)
+
+    *   -   Space / index names
+        -   Yes
+        -   Yes
+        -   Yes
+
+    *   -   Access tuple fields by names
+        -   Yes
+        -   Yes
+        -   Yes
```

Comment:
A simple test does not show it as supported for tarantool-python:

```lua
tarantool> box.cfg{listen = 3301}
tarantool> unpack(box.space.s:format())
---
- {'type': 'unsigned', 'name': 'id', 'version': 2}
- {'type': 'string', 'name': 'd', 'is_nullable': true}
- {'type': 'map', 'name': 'x', 'is_nullable': true}
...
```

```python
>>> import tarantool
>>> c = tarantool.connect('localhost', 3301)
>>> ts = c.select('s')
>>> t = ts[1]
>>> t
[2]
>>> t.id
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'list' object has no attribute 'id'
>>> t['id']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: list indices must be integers or slices, not str
```

AFAIR asynctnt has a tuple object and so likely supports it (but I didn't check).

Reply:
You are right! Thanks, fixed.
```diff
@@ -134,7 +137,7 @@ Last update: January 2022
         -   Yes
 
     *   -   Access tuple fields by names
-        -   Yes
+        -   No
         -   Yes
         -   Yes
 
```

Reply:
gtarantool is based on tarantool-python and likely the same?

Reply:
Yes, set for it 'No' too.